### PR TITLE
[INTERNAL] Improve handling of apps with no namespace configuration

### PR DESCRIPTION
**Pull request by RandomByte**

### CODE REVIEW

File: lib/types/application/ApplicationBuilder.js [hunk 1]
```diff
@@ -16,7 +17,13 @@ const tasks = { // can't require index.js due to circular dependency
 };
 
 class ApplicationBuilder extends AbstractBuilder {
-	addStandardTasks({resourceCollections, project}) {
+	addStandardTasks({resourceCollections, project, log}) {
+		if (!project.metadata.namespace) {
+			log.info("Skipping some tasks due to missing application namespace information. If your project contains" +
+					"a Component.js, you might be missing a manifest.json file. " +
```
Comment (matz3) [inline lib/types/application/ApplicationBuilder.js]: Missing space in the log text "contains**a** Component.js"

File: lib/types/application/ApplicationFormatter.js [hunk 2]
```diff
@@ -20,17 +20,13 @@ class ApplicationFormatter extends AbstractFormatter {
...
 
 			return this.readManifest(project).then(function(manifest) {
-				if (!manifest["sap.app"]) {
-					log.verbose("No 'sap.app' configuration found in manifest of project %s", project.metadata.name);
-					return;
-				}
-				if (!manifest["sap.app"].id) {
-					log.verbose("No application id found in manifest of project %s", project.metadata.name);
+				if (!manifest["sap.app"] || !manifest["sap.app"].id) {
+					log.warn(`No "sap.app" ID configuration found in manifest of project ${project.metadata.name}`, );
```
Comment (matz3) [inline lib/types/application/ApplicationFormatter.js]: Trailing comma

File: lib/types/application/ApplicationBuilder.js (hunk 1 above)
Comment (RandomByte) [reply lib/types/application/ApplicationBuilder.js]: Done

File: lib/types/application/ApplicationFormatter.js (hunk 2 above)
Comment (RandomByte) [reply lib/types/application/ApplicationFormatter.js]: Done

**Pull request merged**